common/scaffolding/reports2/text-level-en.html.html

```diff
@@ -0,0 +1,151 @@
+
+<html>
+<body>
+<nav>
+<b> (a element)</b>
+ <ul>
+  <li> <a href="/">aliquid</a> </li>
+  <li> <a href="/news">News</a> </li>
+  <li> <a>Examples</a> </li>
+  <li> <a href="/legal">Legal</a> </li>
+ </ul>
+</nav>
```

Comment:
This don't seem to be a navigation section. Please remove it

```suggestion
<b> (a element)</b>
 <ul>
  <li> <a href="/">aliquid</a> </li>
  <li> <a href="/news">News</a> </li>
  <li> <a>Examples</a> </li>
  <li> <a href="/legal">Legal</a> </li>
 </ul>

```